src/ches/emailHandlersPublic.ts

```diff
@@ -77,16 +82,19 @@ export const sendProvisionedEmails = async (request) => {
           ]
             .filter(Boolean)
             .map(({ email }) => email),
-          from: "Registry <cloud.pathfinder@gov.bc.ca>",
+          from: 'Registry <cloud.pathfinder@gov.bc.ca>',
           subject:
-            "Your provisioning request for AWS Platform has been completed",
+            'Your provisioning request for AWS Platform has been completed',
         });
       }
-      if (type === RequestType.Edit) {
+      if (
```

Comment:
I believe we can use `else if` instead of `if` here as there is another condition `type === RequestType.Create` above.